src/content/blog/future-of-astro-zero-js-view-transitions.mdx

```diff
@@ -0,0 +1,106 @@
+
+I'm excited to announce that that day has finally arrived. Chrome 126 (released last week) and Microsoft Edge 126 (releasing this week) both support "cross-document view transitions" unlocking **zero-JavaScript view transitions** in Astro.
+
+## Demo
+
+Both Chrome and Edge offer staggered rollout of new versions, so check that your browser is running 126 before running the demo. In chrome, you can see your current version at `chrome://settings/help`.
+
+Open the [astro-zerojs-transitions.vercel.app](https://astro-zerojs-transitions.vercel.app/) demo to experience zero-JavaScript view transitions in your browser. You can explore your browser's Network Panel to verify that every link click triggers a full page load, and zero JavaScript is being sent to the browser to run these animations. If your browser supports it, the demo should look like this:
+
+<figure  class="my-10">
+  <video autoplay loop muted playsinline class="border border-neutral-600 rounded">
+    <source src="/assets/blog/future-of-astro/spotify-demo.mp4" type="video/mp4" />
+    Sorry, your browser doesn't support embedded videos.
+  </video>
+  <figcaption class="text-center text-sm my-4">
+    <a href="https://x.com/astrodotbuild/status/1698534383970656497">Spotify clone</a> built with Astro View Transitions
+  </figcaption>
+</figure>
+ 
+If your browser does not support the latest cross-domain view transitions, then your website will still run and display content as expected. However, the animations will be disabled and page navigation will behave like any other navigation in your browser (without animation).
+
+## Upgrade
+
+To switch to the native new View Transition API, it’s as simple as adding the `@view-transition` at-rule to your website. Here’s all the CSS you need:
+
+```css
+@view-transition {
+  navigation: auto;
+}
+```
+
+To enable cross-document view transitions, both "ends" of the transition must include this rule. A shared page layout component can be a good place to add this to your codebase.
+
+If you were already using Astro View Transitions powered by our `<ViewTransitions />` component, then you are now safe to remove that component as well. This will remove the injected backwards-compatible JavaScript router from your page, reducing the overall complexity of your client-side code.
+
+
+```diff
+---
+- import { ViewTransitions } from "astro:transitions";
+---
+<head>
+-  <ViewTransitions fallback="none" />
++  <style>
++    @view-transition {
++      navigation: auto;
++    }
++  <style>
+</head>
+```
+
+If you would still like your view transitions to run in older browsers, you are also welcome to keep using our backwards-compatible `<ViewTransitions />` component. You can migrate whenever you are ready. We have no plans to remove support for the `<ViewTransitions />` component at this time.
+
+## What about transition:persist?
```

Comment:
If I am not mistaken when we mention pieces of code in headings like this, we tend to also use inline code blocks for them.
```suggestion
## What about `transition:persist`?
```

Reply:
Ah, I wasn't sure if we still used them in headings, since they can look a bit weird. I'm fine with either!